src/main/java/com/bandwidth/iris/sdk/model/RemoveImportedTnOrder.java

```diff
@@ -24,7 +24,6 @@ public class RemoveImportedTnOrder extends BaseModel{
     @XmlElement(name = "LastModifiedDate")
     private String lastModifiedDate;
     @XmlElementWrapper(name = "TelephoneNumbers")
-    @XmlElement(name = "TelephoneNumber")
```

Comment:
Seems to be very specific change. Can we explore/discuss Testing around this?

Reply:
@bpateldx This PR isnt ready yet - i will include a test. Meant to open as draft